### PR TITLE
New/create task

### PR DESCRIPTION
**Pull request by Burbinox**

### CODE REVIEW

**Review by patrykbrodziak1 — APPROVED** [review]

File: server/routes/task.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import express from "express";
+import TaskControler from "../controllers/task.js";
+const router = express.Router();
+
+router.post("/", async (req, res) => {
+  try {
+    const response = await TaskControler.createTask(req.body);
```
Comment (pastepi) [inline server/routes/task.js]:
```suggestion
    const response = await TaskController.createTask(req.body);
```

File: server/routes/task.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import express from "express";
+import TaskControler from "../controllers/task.js";
```
Comment (pastepi) [inline server/routes/task.js]:
```suggestion
import TaskController from "../controllers/task.js";
```

**Review by pastepi — CHANGES_REQUESTED** [review]
Look at the suggestions above.
Also, in files that use `process.env...` please put in this `import "dotenv/config";`
Other than that, this works (I assume we don't want any validation at this point?).

**Comment by Burbinox** [comment]
> Look at the suggestions above. Also, in files that use `process.env...` please put in this `import "dotenv/config";` Other than that, this works (I assume we don't want any validation at this point?).

Why should I? I have PORT in my system variables so I don't need to import "dotenv/config". Same with the DATABASE_PASSWORD.

**Comment by pastepi** [comment]
> > Look at the suggestions above. Also, in files that use `process.env...` please put in this `import "dotenv/config";` Other than that, this works (I assume we don't want any validation at this point?).
> 
> Why should I? I have PORT in my system variables so I don't need to import "dotenv/config". Same with the DATABASE_PASSWORD.

Resolved in DMs, ended up adding the import.

**Review by pastepi — APPROVED** [review]

**Pull request merged**